### PR TITLE
fix/CCN3-1746/returned-health-application-skips-hospital-admissions-page

### PR DESCRIPTION
**Pull request by proctorj**
## Description

Added logic that checks the arc flag status of the other person health check and redirects them to the summary if it is flagged. 

## Todo's before PR

- [x] Branch has been rebased against develop
- [x] Unit tests passed (`make test`)
- [x] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assignees (those who'll merge)
- [x] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/views/other_people_health_check/dob_auth.py [hunk 1]
```diff
@@ -22,10 +22,16 @@ def post(self, request=None):
         """
         adult_id = self.request.GET.get('person_id')
         adult_record = AdultInHome.objects.get(pk=adult_id)
+        application_id = adult_record.application_id_id
+        application = Application.objects.get(application_id=application_id)
 
-        if adult_record.health_check_status == 'COMPLETED' or adult_record.health_check_status == 'FURTHER_INFORMATION':
```
Comment (Thomas-95IS) [inline application/views/other_people_health_check/dob_auth.py]: What happens now in the event that the adult has completed the health check?

Comment (proctorj) [reply application/views/other_people_health_check/dob_auth.py]: Yep, reminder to myself that I need to add 'or adult_record.health_check_status == 'COMPLETED''

Comment (proctorj) [reply application/views/other_people_health_check/dob_auth.py]: @Thomas-95IS Is it possible to access the health check questions if the person has already completed them, other than if they are flagged?

Comment (proctorj) [reply application/views/other_people_health_check/dob_auth.py]: @Thomas-95IS Should be fixed

**Pull request merged**